### PR TITLE
Add sentry Appender

### PR DESCRIPTION
**Pull request by bennsimon**

### CODE REVIEW

File: assets/config/log4j2.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        </logger>
+
+        <logger name="DRISHTI_WEB_MONITOR">
+            <!--<appender-ref ref="WebMonitorLog"/>-->
+            <appender-ref ref="Console"/>
+        </logger>
+
+        <Root level="info">
+            <AppenderRef ref="Console"/>
+            <AppenderRef ref="Sentry"/>
```
Comment (githengi) [inline assets/config/log4j2.xml:46]: this will send even info logs

File: assets/config/log4j2.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        <File name="ReportingMonitorLog" fileName="opensrpReportingMonitoring.log">
+            <PatternLayout pattern="%d{ddMMyyyy HH:mm:ss,SSS} %m%n"/>
+        </File>
+        <File name="WebRollingLog" fileName="opensrpWeb.log">
+            <PatternLayout pattern="%d{ddMMyyyy HH:mm:ss,SSS} %-5p [%t] [%c{1}] %m%n"/>
+        </File>
+        <File name="WebMonitorLog" fileName="opensrpWebMonitoring.log">
+            <PatternLayout pattern="%d{ddMMyyyy HH:mm:ss,SSS} %m%n"/>
+        </File>
+        <Sentry name="Sentry"/>
```
Comment (githengi) [inline assets/config/log4j2.xml:19]: are we including all lo levels?

Comment (githengi) [reply assets/config/log4j2.xml:19]: also not sure how is the connection to sentry defined, what happens to dev environments where we may not want to send logs to sentry.

File: assets/config/log4j2.xml (hunk 1 above)
Comment (bennsimon) [reply assets/config/log4j2.xml:46]: Nope by default Sentry sends `Error` and above

File: assets/config/log4j2.xml (hunk 2 above)
Comment (bennsimon) [reply assets/config/log4j2.xml:19]: Nope by default Sentry sends `Error` and above. If the `dsn` is an empty string then the Sentry SDK is closed

File: assets/config/log4j2.xml (hunk 1 above)
Comment (bennsimon) [reply assets/config/log4j2.xml:46]: https://docs.sentry.io/platforms/java/guides/log4j2/#configure

File: assets/config/log4j2.xml (hunk 2 above)
Comment (bennsimon) [reply assets/config/log4j2.xml:19]: https://docs.sentry.io/platforms/java/guides/log4j2/#configure

**Review by githengi — APPROVED** [review]

**Pull request merged**